plugins/modules/site_workflow_manager.py

```diff
@@ -698,19 +893,33 @@ def site_requires_update(self):
             specified parameters, such as the site type and site details.
         """
 
-        type = self.have['current_site']['type']
-        updated_site = self.have['current_site']['site'][type]
-        requested_site = self.want['site_params']['site'][type]
+        if self.get_ccc_version_as_integer() <= self.version_2_3_5_3:
+            type = self.have.get('current_site', {}).get('type')
+            updated_site = self.have.get('current_site', {}).get('site', {}).get(type)
+            # requested_site = self.want['site_params']['site'][type]
+            self.log(self.pprint(self.want))
```

Comment:
don't include things like pprint and log self.want in the def get_want API only.

Reply:
Removed